tests/network/user_defined_network/ip_specification/libipspec.py

```
@@ -47,7 +47,7 @@ def read_guest_interface_ipv4(
         IpNotFound: If no IPv4 address is found on the specified interface.
     """
     cmd: Final[str] = f"ip -j -4 addr show {interface_name}"
-    if not (out := vm.console(commands=[cmd], timeout=10)):
+    if not (out := vm.console(commands=[cmd], timeout=10))[cmd]:
         raise RuntimeError(f"Failed to retrieve IP address from {interface_name}")
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)